README.md

```diff
@@ -3,13 +3,13 @@
 ![problems-solved](https://img.shields.io/badge/Problems%20Solved-21/284-1f425f.svg)
```

Comment:
you also need to increment total problems solved :) 

PS. keep up the good work, keep them coming 👍

Reply:
Done with correction